### DISCUSSION TITLE
My application gets instantly rejected.

### DISCUSSION THREAD

**Discussion by pimglar** [community/community#49592] [GitHub Education]
Hi, i have a problem with my application. I reached before to github support but they did not give me any solution, so I'm asking here in hope that someone had the same problem as me. I 
![33C1B9CD-4840-412A-9A05-6653F887DD94](https://user-images.githubusercontent.com/119127477/223939708-da4a9fd9-a1d5-4a5f-9db2-1e64ffdcfbe9.png)
![A9B3B503-C37E-4D62-BC37-C09172E51683](https://user-images.githubusercontent.com/119127477/223939731-feceb3d8-cd43-4560-9b7f-b159d0bbc87a.png)
I don't know what could be an issue... I have my full name, and a short bio + I submitted with my valid and translated school ID.

**Comment by pimglar** [comment]
I tried to contact GitHub staff and the education team with no success. All of them told me to open a disscusion on this community... I'm so confused right now because I can't get an answer. "Individual manual reviews and status updates are no longer provided. " This is so frustrating..

**Reply by pimglar** [reply]
I tried to email them, but they now require contacting them via Support website. And I failed to get any reasonable response from them... I don't think I made any mistakes in my information, and I made sure that my school's ID is clear and is displaying properly.
Although, maybe it's auto-rejecting because my full name on my school ID is handwritten and therefore perhaps is unable to read, but if so, I don't know how I can prove them that this is my real info...

**Reply by ghost** [reply]
> I tried to email them, but they now require contacting them via Support website. And I failed to get any reasonable response from them... I don't think I made any mistakes in my information, and I made sure that my school's ID is clear and is displaying properly. Although, maybe it's auto-rejecting because my full name on my school ID is handwritten and therefore perhaps is unable to read, but if so, I don't know how I can prove them that this is my real info...

I understand how frustrating it can be to receive an automatic rejection without any clear explanation. It's possible that the handwriting on your school ID could be causing the issue, but it's difficult to say for certain without any further information.

One thing you could try is to reach out to your school and ask if they have a digital copy of your ID or if they could provide you with a typed version of your full name. This could help to eliminate any confusion or errors that may have arisen due to the handwritten nature of your ID.

Additionally, if you haven't already, you could try submitting a support ticket on the GitHub Support website explaining your situation and including any additional information or documentation that could help to verify your identity. It may take some time to receive a response, but it's worth trying in case there's a way to resolve the issue.

Remember to be patient and persistent in your efforts. It's possible that the issue is just a minor error or misunderstanding that can be easily resolved with a bit of extra effort. Good luck, and don't give up!

**Reply by pimglar** [reply]
I'm done (at least for now) with the support team… They told me to find answer here, so maybe If I patiently wait I can get a response from education moderator on this site to help me.

**Reply by pimglar** [reply]
> Regarding the issue with your handwritten name on your school ID, you can try providing additional documentation to prove your identity, such as a government-issued ID or a copy of your birth certificate. You can also try contacting your school to see if they can provide a typed version of your name or a letter confirming that the handwriting on your school ID is indeed your name. You can try something quick like putting your National ID beneath the School ID and upload the photo so that you prove both your identity and affiliation to the program

I still need to know the reason for the rejection, but you have a good idea.

**Reply by WilliamLLee** [reply]
I  have the same problem, and had not fixed till now!

**Reply by pimglar** [reply]
Let's just hope that they are going to address this issue fast

**Comment by ghost** [comment]
I'm sorry to hear that you're having trouble with your application. It's understandable that you're frustrated with the lack of response from the GitHub staff. It's possible that there could be an issue with the information you provided or something else that's causing your application to be rejected.

If you haven't already, you could try providing more detailed information in your application and ensuring that all of the information you provide is accurate and up-to-date. It might also be helpful to reach out to other individuals or organizations who have successfully applied for the education program and ask for their advice or guidance.

Unfortunately, since GitHub no longer provides individual manual reviews or status updates, it might be difficult to determine the exact reason why your application is being rejected. However, don't give up hope! Keep trying to reach out to the GitHub staff and education team and continue to refine and improve your application. Good luck!

**Reply by pimglar** [reply]
Thanks for the reply!
Maybe it's auto-rejecting because my full name on my school ID is handwritten and therefore perhaps is unable to read, but if so, I don't know how I can prove them that this is my real info...

**Reply by ghost** [reply]
I can understand how frustrating it can be to have your application rejected without any clear explanation. It's possible that the handwritten name on your school ID is causing issues with the automated review process. One solution could be to try providing an alternative form of identification that has your name printed clearly, such as a passport or driver's license.

If you don't have access to an alternative form of identification, you could try reaching out to your school and asking if they can provide a digital copy of your ID with your name printed clearly. This could help provide additional evidence that the name on your handwritten ID is indeed your real name.

Additionally, it might be helpful to provide more information in your application about your academic history and goals for using GitHub. This could help demonstrate your eligibility for the education program and increase your chances of being accepted.

I understand that the lack of communication from the GitHub staff can be frustrating, but don't give up hope. Keep trying to reach out to them and explore alternative solutions. With some persistence and effort, I'm confident that you'll be able to successfully apply for the education program.
